gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/HiveSource.java

```diff
@@ -230,6 +220,25 @@ private void createWorkunitsForPartitionedTable(HiveDataset hiveDataset,
     }
   }
 
+  @VisibleForTesting
+  public boolean shouldCreateWorkunitForPartition(Partition partition, long updateTime, LongWatermark lowWatermark) {
```

Comment:
You don't need partition as param for shouldCreateWorkUnitForPartition(), and when you remove that shouldCreateWorkUnitForPartition() and shouldCreateWorkUnitForTable() become exactly the same, so they can merged to be named shouldCreateWorkUnit()

Also, not a strong preference however: I liked that you had isOlderThanLookback() as part of the logic in shouldCreateWorkUnitForPartition() earlier. But feel free to keep it as is or as earlier